org.torproject.torbrowser-launcher.yaml

```diff
@@ -0,0 +1,217 @@
+app-id: org.torproject.torbrowser-launcher
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: 5.15-23.08
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+build-options:
+  env:
+    - BASEAPP_REMOVE_WEBENGINE=1
+separate-locales: false
+command: torbrowser-launcher
+rename-desktop-file: torbrowser.desktop
+rename-icon: torbrowser
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --talk-name=org.freedesktop.Notifications
+  - --device=dri
+  - --socket=pulseaudio
+  - --filesystem=xdg-download
+  - --env=MOZ_ENABLE_WAYLAND=1
```

Comment:
Why not use the ESR default?

Reply:
As far as I know, Tor Browser still defaults to not enabling Wayland by default. This is however not what we want in the Flatpak'ed version.

Reply:
Would be better to follow the upstream default, imo, but up to you